samples/zigbee/shell/README.rst

```diff
@@ -0,0 +1,166 @@
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-rows:: /includes/sample_board_rows.txt
+   :header: heading
+   :rows: nrf52840dk_nrf52840, nrf52833dk_nrf52833, nrf5340dk_nrf5340_cpuapp, nrf21540dk_nrf52840
+
+You can use two or more of the development kits listed above and mix different development kits.
+
+Additionally, you can use this sample with any other :ref:`Zigbee <ug_zigbee>` sample application for testing the communication or other sample-specific functionalities.
+
+Overview
+********
+
+The Zigbee shell sample takes the Zigbee Router role and implements two clusters (Basic and Identify) that used to be required by the Zigbee Home Automation profile.
+The Basic cluster provides attributes and commands for determining basic information about the node.
+The Identify cluster allows to put the device into the identification mode, which provides a way to locate the device.
+The device also includes all Zigbee shell commands, that allow for discovering, controlling and testing other Zigbee devices.
+
+User interface
+**************
+
+LED 3:
+    Turns on when the device joins the network.
+
+LED 4:
+    Blinks to indicate that the identification mode is on.
+
+Button 4:
+    Toggles the identification mode on the device.
+
+All other interactions with the application can be handled using serial communication.
+See :ref:`zigbee_cli_reference` for the list of available serial commands.
+
+Building and running
+********************
+.. |sample path| replace:: :file:`samples/zigbee/shell`
+
+|enable_zigbee_before_testing|
+
+.. include:: /includes/build_and_run.txt
+
+.. _zigbee_shell_sample_testing:
+
+Testing
+=======
+
+After building the sample and programming it to your development kits, test it by performing the following steps:
+
+#. Turn on the development kits.
+#. Set up the serial connection with the development kits that are programmed with the Zigbee CLI sample.
+   You can use one of the following CLI interfaces:
+
+   * UART - For more details, see :ref:`putty`.
+   * RTT - For more details, see :ref:`testing_rtt`.
+
+#. Depending on whether you have access to a standalone Zigbee Coordinator:
+
+   * If you have a standalone Zigbee Coordinator, make sure it is able to accept new nodes.
+   * If you do not have a standalone Zigbee Coordinator, use one of the following solutions:
+
+     - Use a Zigbee Coordinator from the :ref:`zigbee_network_coordinator_sample` sample and flash it on a third board.
+     - Set up one of the CLIs to act as Zigbee Coordinator (see `Setting up CLI device as Zigbee Coordinator`_).
+
+#. By default, the CLI sample implements the Zigbee Router functionality.
+   If you have a coordinator nearby that is using channel 16, run the following command on boards programmed with the CLI sample (excluding the coordinator board, if you use a CLI board as coordinator):
```

Comment:
```suggestion
#. Given that by default the CLI sample implements the Zigbee Router functionality, if you have a coordinator nearby that is using channel 16, run the following command on boards programmed with the CLI sample (excluding the coordinator board, if you use a CLI board as coordinator):
```
We'll need to rework this CLI board as coordinator in the docs, as it's getting messy, but we do this later.

Reply:
If we restructure the testing scenario, this will be improved too.